bigflow/bigquery/__init__.py

```diff
@@ -1,13 +1,13 @@
-
 try:
     from .interactive import interactive_component as component
-    from .interactive import add_label_component, sensor_component, INLINE_COMPONENT_DATASET_ALIAS
+    from .interactive import add_label_component, sensor, INLINE_COMPONENT_DATASET_ALIAS
```

Comment:
Maybe from `add_label_component` we also colud remove `_component` suffix?

Reply:
done